exercise/staff_views.py

```diff
@@ -202,6 +205,97 @@ def get_common_objects(self):
         )
 
 
+class ExamSessionEdit(UpdateView):
+    template_name = 'exammode/staff/edit_session_form.html'
+    form_class = ExamSessionForm
+    #fields = ['exam_module', 'can_start', 'duration', 'room']
+    model = ExamSession
+
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs['course_instance'] = self.get_object().course_instance
+        return kwargs
+
+    '''
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        instance = self.get_object().course_instance
```

Comment:
This commented out `__init__` method should not be here, I suppose?

Reply:
The comment is marked as 'Outdated* since the staff_views.py file has been removed and the classes moved to the views.py. The commented init function is still there.